float32 pos_x
float32 pos_y
---
int32 goal_feedback


